MacaoProject/Macao/Models/Deck.cs

```diff
@@ -1,25 +1,53 @@
 ﻿using Macao.Enums;
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.TextBox;
 
 namespace Macao
 {
     public class Deck
     {
         public List<Card> Cards { get; set; } = new List<Card>();
 
-        public void Move(Deck discardDeck)
-        {
 
+        public void Shuffle(List<Card> shuffleCards)
```

Comment:
this method should shuffle it's own cards not cards received as a parameter. think about how you shuffle a deck by yourself, you do not use another set of cards (or deck) to do this.